build/params_2k.go

```diff
@@ -120,3 +120,17 @@ const InteractivePoRepConfidence = 6
 const BootstrapPeerThreshold = 1
 
 var WhitelistedBlock = cid.Undef
+
+const DelegatedPoWFinality = 2
```

Comment:
Why not using a map instead a list of consts in the end?

Reply:
We will get cycles.